### DISCUSSION TITLE
What is the spaCy 3 equivalent of the following code?

### DISCUSSION THREAD

**Discussion by romanRegmi** [explosion/spaCy#10321] [Help: Coding & Implementations]
I am trying to follow a tutorial that was made using spaCy v2. It turns out that with the update the whole training process changed. Now I cannot figure out the changes I have to make in the code. 

```Python
import spacy

nlp = spacy.blank("en")
ner = nlp.create_pipe('ner')
ner.add_label('CONC_CAMP')

nlp.add_pipe(ner, name='conc_camp_ner')
nlp.to_disk('holocaust_ner')
```
Here is what I have tried.

```Python
import spacy

nlp = spacy.blank("en")


nlp.add_pipe("ner", name='conc_camp_ner')

nlp.to_disk("holocaust_ner")

ner = nlp.get_pipe('conc_camp_ner')

ner.add_label('CONC_CAMP')
```
The code is supposed to create a label in the meta.json file. But it fails to do so. 
```JSON
"labels":{
    "conc_camp_ner":[
      "CONC_CAMP"
    ]
  }
```
But the label is not created in my case.
```JSON
"labels":{
    "conc_camp_ner":[
    ]
  }
```

**Comment by ljvmiranda921** [comment]
> It turns out that with the update the whole training process changed

Hi @romanRegmi, if you're training an NER model in v3, then you should do it [through the spaCy config](https://spacy.io/usage/training#quickstart). That means you don't have to write any code for training, although you have to convert your text [into a serialized format](https://spacy.io/api/docbin). You can check a few examples in the [spaCy projects repo](https://github.com/explosion/projects/tree/v3/pipelines).
